crates/ef-testing/src/lib.rs

```diff
@@ -1,4 +1,5 @@
 pub mod models;
+pub mod sequencer;
```

Comment:
there is a sequencer crate, imported in the ef-testing crate, but the ef-testing crate also has a sequencer module? this is a bit confusing

Reply:
want me to rename to `evm-sequencer`?